src/Unleash/Scheduling/FetchFeatureTogglesTask.cs

```diff
@@ -56,7 +56,7 @@ public async Task ExecuteAsync(CancellationToken cancellationToken)
             }
             catch (HttpRequestException ex)
             {
-                Logger.Error(() => $"UNLEASH: Unhandled exception when fetching toggles.", ex);
+                Logger.Warn(() => $"UNLEASH: Unhandled exception when fetching toggles.", ex);
```

Comment:
This will remove the error log output during synchronous init however

Reply:
We could do a check here as well and log as error if it's during synch init

Reply:
This is definitely a smell at this point, checking who the caller is so that we can log at the right level just means that the exception is being handled at the wrong level

I think this is good enough for the thing that we want to solve. If we want this logging to work rationally, I think this needs some overhaul, which is definitely a bigger thing (and belongs in another PR)